source/technical/daypass.rst

```diff
@@ -0,0 +1,91 @@
+.. _daypass:
+
+=======
+Daypass
+=======
+Normally, only Chameleon users with active allocations are able to launch and
+view Trovi artifacts. To allow anyone to launch an artifact, we also provide
+daypass. This allows for a non-Chameleon user to have access to Chameleon
+for a limited amount of time, using a small, separate allocation.
+People interested in reproducing your project will send requests
+to the managers of a project. If approved, the requesting user will recieve an
+email invitation to join a reproducibility project. When they accept, they
+can use this project to run your artifact. After the specified time limit,
+they will be automatically removed from this project.
+
+.. _enable-daypass:
+
+Allowing Reproducibility Requests
+=================================
+
+First, the owner of an artifact must permit reproducibility requests. This can
+be revoked at any time, preventing future requests. Additionally, you must also
+give your artifact a value for "Hours a user has to reproduce." This value
+specifies how long a user will have access to Chameleon for. Consider how
+long it takes to run your experiment from start to finish as a lower bound for
+this value. The artifact owner must also assign their artifact to a project via
+the dropdown selector. As these requests are granting access to Chameleon
+resources, this is needed to tie granted requests to a PI.
+
+These fields can be accessed by navigating to an artifact's detail page, and
+then selecting "Share." At the bottom of the share page, you will see the
+below forms, which are the project assignment, the enabling of reproducibility
+requests, and the hours to reproduce.
+
+.. figure:: daypass/sharing_reproducibility.png
+   :alt: An image showing the sharing fields for reproducibility requests
+   :figclass: screenshot
+
+After these items are saved, an allocation request is automatically made. Your
+artifact should now appear with a "Request Daypass" button below the "Launch"
+button. The "Launch" button will not appear for users that are not a member
+in an active Chameleon project.
+
+Requesting a Daypass
+--------------------
+
+When you select "Request Daypass", you will be taken a form where you are asked
```

Comment:
The subject for each section is a bit different (PI/manager versus general researcher), yet the second person is always used. I'm not quite sure how best to resolve this so it's more clear that we're jumping b/w the different use-cases here.

Maybe "When another researcher wishes to reproduce your artifact (or when you wish to do so for another's artifact), selecting "Request Daypass" will display a form asking for a name, institution, and the reason for reproducing the artifact. For artifacts under your project/allocation, this gives you oversight and discretion as to who you grant access to, as ultimately you're responsible for usage under your allocation (even reproducibility "sub-allocations".) For artifacts you wish to explore, it gives you the chance to reach out to the project owners and explain why you are interested in their work."